tools/vagrant/provision-vagrant.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 set -e
-VERSION="1.9.2"
+VERSION="1.11.2"
```

Comment:
Any particular reason you picked 1.11.x? With go module support, we may want a later version and I know KinD is saying 1.12.5 or later...

Reply:
1.12.5 is the latest stable version.  I have tested with that and re-pushed.